converter/convertBIN2csv.m

```diff
@@ -23,34 +30,46 @@
 
 %% Reading file
 while read
-    packet = parsePacket(fid);
+    packet = parseBtPacket(fid);
     switch packet.type
         case 'orn'
             ORN.data = cat(2, ORN.data, packet.orn);
             ORN.timestamp = cat(2, ORN.timestamp, packet.timestamp);
-            
         case {'eeg4', 'eeg8'}
+            nSample = size(packet.data, 2);
+            t = linspace(packet.timestamp, packet.timestamp + ...
```

Comment:
is this time? not sure what's happening, a small comment would be good! e.g. % create list of timestamps for packet